spyderlib/plugins/shortcuts.py

```diff
@@ -176,6 +138,7 @@ def __init__(self, parent, context, name, sequence, shortcuts):
         self.text_new_sequence = CustomLineEdit(self)
         self.text_new_sequence.setPlaceholderText(sequence)
         self.helper_button = HelperToolButton()
+        self.label_warning = QLabel('<br><br>')

```

Comment:
Ok I will add only one break, otherwise the dialog will resize if any error is displayed below....